_includes/tinydrive-script-tag.html

```diff
@@ -29,4 +29,4 @@
   margin-right: 3px;
 }
 </style>
-<script src='{{ site.tinydrive_codepen_url }}'></script>
+<script src='{{ site.tinydrive_live-demo_url }}'></script>
```

Comment:
Some more inconsistent naming here.

Reply:
Fixed in https://github.com/tinymce/tinymce-docs/pull/1645/commits/50d665fdbcdb9afd3395dccf0ff6eba409f4895d